db/migrate/20180131114516_change_dimensions_items_temps_fields.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDimensionsItemsTempsFields < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :dimensions_items_temps, :link, :base_path
```

Comment:
You're no longer using the `dimensions_items_temps` table so it should be dropped. (See the deletion of `TemporaryItemStore`.